jobs/consul_agent_windows/monit

```diff
@@ -0,0 +1,9 @@
+{
+  "processes": [
+    {
+      "name": "consul",
+      "executable": "C:\\var\\vcap\\packages\\confab-windows\\bin\\confab.exe",
+      "args": [ "start", "", "-config-file", "/var/vcap/jobs/consul_agent_windows/confab.json", "-foreground" ]
+    }
+  ]
```

Comment:
Consul has a hard-coded DNS timeout of 2s, which is very bad in some environments.  We see this being especially problematic in China.

Consul also has issues handling large DNS response payloads.

Both these issues are critical and the Infrastructure team is has been implementing putting in dnsmasq recently, as a top priority.